.travis.yml

```diff
@@ -2,17 +2,18 @@ language: python
 sudo: false
 matrix:
   include:
-  - python: '3.5'
-    env: TOX_ENV=py35,flake8
+  - python: '3.7'
+    env: TOXENV=flake8
   - python: '3.6'
-    env: TOX_ENV=py36
+  - python: '3.7'
+  - python: '3.8'
```

Comment:
These can be defined at the `language` section of travis. See [graphql-server-core#35 travis](https://github.com/graphql-python/graphql-server-core/blob/d8b98b7342bc4a451ffb324082f4f8dd3b9397e1/.travis.yml) file.